h/schemas/annotation.py

```diff
@@ -311,3 +311,88 @@ def _target_selectors(targets):
         return targets[0]['selector']
     else:
         return []
+
+
+class SearchParamsSchema(JSONSchema):
+    """Validates a JSON payload for an elasticsearch query via API"""
+    schema = {
+        # 'type': 'object',
+        'properties': {
```

Comment:
We need to have a discussion/come to an agreement on these/aka which fields we want to make query-able. I have eliminated some I felt we pretty obviously didn't want to expose in a particular commit (of course there may be some discussion on those too).

Reply:
Once we agree on these then I will add them to the api docs.